static/src/javascripts/bootstraps/video-embed.js

```diff
@@ -106,9 +175,14 @@ define([
 
                 player.fullscreener();
 
+                deferToAnalytics.init();
```

Comment:
this won't do anything useful, I think I meant somehow cause the module deferToAnalytics to be inited in such a way that it cannot miss the event it's listening for.

So you can remove the deferToAnalytics reference here